playbooks/tasks/install-python.yaml

```diff
@@ -0,0 +1,53 @@
+# coreos' toolbox allows us to inject our own tooling via a container filesystem and systemd-nspawn
+# we configure toolbox to run a python container, then configure ansible to use this for its modules
+# https://github.com/coreos/toolbox
+
+- name: create toolboxrc
+  # ansible prepends spaces in front of multiline raw commands, breaking heredocs
+  # one workaround for this is to assign the command to a variable as we've done here
+  # https://github.com/ansible/ansible/issues/12034#issuecomment-453723192
+  raw: "{{ cmd }}"
+  vars:
+    # escaping or quoting the "limit string" in a heredoc disables parameter substitution
+    # http://tldp.org/LDP/abs/html/here-docs.html#EX71C
+    cmd: |
+      cat > ~/.toolboxrc << \EOF
```

Comment:
Consider linking this file `/root/.toolboxrc` so this config can be sourced by toolbox as root.

Reply:
Toolbox unpacks the container filesystem to a [directory containing the user name](https://github.com/coreos/toolbox/blob/586dcee184afe5f9d4a5b27478132e7b0f7d1173/toolbox#L45-L46) so it's not that simple. Fortunately though we can symlink that directory too!